public/js/sortfilter.js

```diff
@@ -207,6 +209,79 @@ function filterApply(card, filter) {
       }
     }
   }
+  if(filter.category == 'tag') {
+    card.tags.forEach(function(val, index){
+      if(val.toLowerCase() == filter.arg)
+      {
+        res = true;
+      }
+    });
+  }
+  if(filter.category == 'status') {
+    if(card.status.toLowerCase() == filter.arg)
+    {
+      res = true;
+    }
+  }
+
+  if(filter.category == 'price')
+  {    
+    var price = null;
+    if (card.details.price) {
+      price = card.details.price;
+    } else if (card.details.price_foil) {
+      price = card.details.price_foil;
+    }
+    if (price) {
+      switch(filter.operand)
+      {
+        case ':':
```

Comment:
Not for this PR, but I think we could combine all this code into one switch statement instead of having all the copy-and-pasted code.